tests/integration-tests/tests/dashboard/test_dashboard.py

```diff
@@ -31,17 +31,38 @@ def test_dashboard(
     )
     cluster = clusters_factory(cluster_config)
     cw_client = boto3.client("cloudwatch", region_name=region)
-    dashboard_name = "{0}-{1}".format(cluster.cfn_name, region)
 
+    # test CWAgent metrics
+    metrics_response = cw_client.list_metrics(Namespace="CWAgent")
```

Comment:
From cloudwatch documentation: After you create a metric, allow up to 15 minutes for the metric to appear. To see metric statistics sooner, use [GetMetricData](https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_GetMetricData.html) or [GetMetricStatistics](https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_GetMetricStatistics.html).

What about using get-metric-data? In this way we can have a test to verify that we have disk and mem metric data for headnode, but we don't have that data for compute nodes

Reply:
OK will work on this